CI_JobHistory.py

```diff
@@ -241,9 +248,11 @@ def main():
     parser = argparse.ArgumentParser(description='Get the job history')
     parser.add_argument('--zone', help='specify the lease/zone', type= lambda arg:arg.split(','))
     parser.add_argument('--job_type', default='p', choices=['p','z','pa'], help= 'Specify the CI job type (Power(p) or s390x(z) or Power Auxillary(pa)), default is p')
+    parser.add_argument('--job_platform',default=' ', help='enter job_platform as heavy libvirt powervs upgrade')
```

Comment:
@alokgoswami-ag Can you add a meaningful default value instead `' '`.

Reply:
i have added default value with 'ALL'